backend/src/director.py

```diff
@@ -10,9 +11,13 @@
 director_prompt = engine.load_prompt("director")
 determine_intention_prompt = engine.load_prompt("determine-intention")
 
+
 def question(question):
```

Comment:
Could we improve the naming of this function?

Reply:
hah my bad for not reading this comment before replying to the previous one! yes, I think in the next PR